services/terrain/src/terrain/util/config.clj

```diff
@@ -502,6 +502,8 @@
        (cfg/env-setting "TREE_URLS_PORT")
        (tree-urls-base-url)))))
 
+(defn tree-urls-attr [] "ipc-tree-urls")
```

Comment:
I changed this to be configurable as well as changed it to be an administrative AVU (starting with ipc-). This value serves only as a cache and wasn't even being properly fetched before, so this has no downsides other than potentially leaving around some unused 'tree-urls' (the previous AVU name) AVUS.